yang/vyatta-protocols-frr-isis-v1.yang

```diff
@@ -0,0 +1,878 @@
+module vyatta-protocols-frr-isis-v1 {
```

Comment:
There seems to be a lot of whitespace errors in this file. For reference, the DANOS convention is to use Tabs to do indentation in YANG files.

Reply:
We have fixed this issue.